src/App.jsx

```diff
@@ -0,0 +1,20 @@
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import SharePage from "./pages/share/SharePage.js";
+import FolderPage from "./pages/folder/FolderPage.js";
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<SharePage />}></Route>
```

Comment:
self-closing 신경써주시면 좋습니다~
```suggestion
        <Route path="/" element={<SharePage />} />
```